### ISSUE TITLE
Whitelist of plugins that can cause false positives, not updated in 3 years

### ISSUE THREAD

**Issue by the-de-coder** [wpengine/phpcompat#246]
"We maintain a whitelist of plugins that can cause false positives."

The list has not been updated in 3 years

**Comment by the-de-coder** [comment]
When viewing the plugin main page, at the bottom of the screen under Limitations & Caveats, the first item is:

“This tool cannot detect unused code paths that might be used for backwards compatibility, potentially showing false positives. We maintain a whitelist of plugins that can cause false positives.”

 

Which links to a 3 year old page.

**Comment by richboos** [comment]
Yeah noticed this too. The plugin points to an outdated page. If it wasn't, I don't believe @the-de-coder would even have noticed the outdated page.

Discount Rules for WooCommerce can be added to the whitelist, allegedly, according to the author. https://wordpress.org/support/topic/failing-some-php-compatibility-checks/#post-13156449
